contracts/test/helpers/ACLHelper.sol

```diff
@@ -3,17 +3,6 @@ pragma solidity 0.4.24;
 import "../../acl/IACLOracle.sol";
 
 
-contract ACLHelper {
-    function encodeOperator(uint256 param1, uint256 param2) internal pure returns (uint240) {
-        return encodeIfElse(param1, param2, 0);
-    }
-
-    function encodeIfElse(uint256 condition, uint256 successParam, uint256 failureParam) internal pure returns (uint240) {
-        return uint240(condition + (successParam << 32) + (failureParam << 64));
-    }
-}
-
-
 contract AcceptOracle is IACLOracle {
```

Comment:
Could we rename this file, since it only contains test `ACLOracle`s now?

Reply:
Renamed it to ACLOracleHelper